docs/platform/porting/graphics-and-ui.md

```diff
@@ -379,6 +416,47 @@ The output backend interface is mandatory. For more information, see [tdm_backen
+| `hwc_get_client_target_buffer_queue()` | Gets a target buffer queue. The backend returns `tbm_surface_queue_h`. | Yes |
+| `hwc_set_client_target_buffer()`       | Sets the client (relative to TDM) target buffer. The target buffer is from `tbm_surface_queue_h` that contains the result of the gl composition with `tdm_hwc_window`. | Yes |
+| `hwc_validate()`                       | Validates HWC. The backend inspects all the hardware layer states and determines whether there are any composition type changes necessary before committing HWC. | Yes  |
+| `hwc_get_changed_composition_types()`  | Gets the changed composition types. The backend returns `tdm_hwc_window` and the `tdm_hwc_window` composition type is changed through `hwc_validate`. | Yes  |
+| `hwc_accept_validation()`              | Accepts the validation required by the backend. The backend can identify the decided `tdm_hwc_window_composition` at the required validation. The backend can commit the set of `tdm_hwc_window` with this accepted `tdm_hwc_window_composition`. | Yes  |
+| `hwc_commit()`                         | Commits changes for an HWC object. The backend can commit output (layers), associated with the accepted validation of the HWC object on the display output device. | Yes  |
+| `hwc_set_commit_handler()`             | Sets a user commit handler. The backend has to call `tdm_hwc_commit_handler` after finishing `hwc_commit`. | Yes  |
+| `hwc_set_property()`                   | Sets the property that has a given property ID by the backend on the HWC object. | Yes  |
+| `hwc_get_property()`                   | Gets the property that has a given property ID by the backend on the HWC object. | Yes  |
+
+The hwc_window backend interface is mandatory. For more information, see [tdm_backend.h](https://review.tizen.org/gerrit/gitweb?p=platform/core/uifw/libtdm.git;a=tree;h=refs/heads/tizen;hb=refs/heads/tizen).
+
+**Table: HWC backend interface functions**
+
+| Function                   | Description                              | Mandatory          |
+| -------------------------- | ---------------------------------------- | --------- |
+| `hwc_window_destroy()`              | Destroys `tdm_hwc_window`. The backend module must implement this function. The backend destroys the private window, tdm_hwc_window. | Yes |
+| `hwc_window_acquire_buffer_queue()` | Acquires a buffer queue associated with `tdm_hwc_window`. This function can be used when the backend has TDM_HWC_WIN_CONSTRAINT_BUFFER_QUEUE. | No  |
+| `hwc_window_release_buffer_queue()` | Releases a buffer queue assoicated with `tdm_hwc_window`. This function can be used when the backend has TDM_HWC_WIN_CONSTRAINT_BUFFER_QUEUE. | No  |
+| `hwc_window_set_composition_type()` | Gets the composition type of `tdm_hwc_window`. The backend sets `tdm_hwc_window_composition`.| Yes |
```

Comment:
In the function name, "set"
In the description. "get"

The description and function name are slightly different. Would you please confirm it?

Reply:
thx I will fix it.